client/src/components/GraphViewerComponent/GraphViewerCytoscapeComponent/GraphViewerCytoscapeComponent.js

```diff
@@ -419,6 +419,10 @@ export class GraphViewerCytoscapeComponent extends React.Component {
     return (colors[(colors.length - 1) - im]);
```

Comment:
Is this still needed?

Reply:
Also maybe could remove the import?

Reply:
It is because there is still color logic related with the relationships